tests/protos/vehicle.proto

```diff
@@ -1,7 +1,7 @@
 // Proto definition for a Car
 
-import "common.proto";
-import "person.proto";
+import "tests/protos/common.proto";
```

Comment:
yep! i added 
1) A parameter to not follow imports
2) imports are now relative to the protoc working directory, rather than the protobuf file (which is what protoc expects)